### ISSUE TITLE
New to TAL, Issue replicating results

### ISSUE THREAD

**Issue by brandonclark314** [frostinassiky/gtad#51]
Hello,

Thank you for providing the code for G-TAD and making it easy to run training and post-processing. I'm relatively new to this field and was attempting to replicate your results before I tried some new experiments. I followed your instructions for installation and used `bash gtad_thumos.sh | tee log.txt` to run everything but my results were lower than what is listed in the README.md, instead I got the results below.

mAP at tIoU 0.3 is 0.4337173950647065
mAP at tIoU 0.4 is 0.38620687412161775
mAP at tIoU 0.5 is 0.31979038731209003
mAP at tIoU 0.6 is 0.24733233931294088
mAP at tIoU 0.7 is 0.1747389841614611

I tried some different things like changing the learning rate and more epochs but nothing got close enough to your results to seem like it was the expected result.

Any help is greatly appreciated.

Thanks,
Brandon Clark
